model-integration/src/main/java/ai/vespa/llm/clients/OpenAI.java

```diff
@@ -9,40 +9,59 @@
 import com.yahoo.api.annotations.Beta;
 import com.yahoo.component.annotation.Inject;
 
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import java.util.concurrent.CompletableFuture;
 import java.util.function.Consumer;
 
 /**
  * A configurable OpenAI client.
  *
- * @author lesters
+ * @author lesters glebashnik
```

Comment:
FYI use multiple lines with `@author` prefix for multiple authors. Javadoc will assume this is a single author ;)